### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by gauravharsha**
The cleanup addresses two main points:

1. Use `verbose` as an input when initializing the integral reader. That way, a single parameter controls the verbosity of the entire Green-MBPT program.
2. Remove legacy files and functions.
3. Cosmetic restructuring in `CMakeLists.txt`

### CODE REVIEW

File: src/green/integrals/df_integral_t.h [hunk 1]
```diff
@@ -129,6 +126,7 @@ namespace green::integrals{
     /**
      * Extract V(Q, i, j) with given (k1, k2) in precision "prec" from the entire integrals (Vk1k2_Qij)
      * TODO: this non-chunked version should be combined with the chunked version
+     * ? GH: This task appears to be accomplished. Or does the TODO refer to something else?
```
Comment (egull) [inline src/green/integrals/df_integral_t.h]: correct. This is done.

File: test/test_buffer.cpp [hunk 2]
```diff
@@ -32,8 +32,12 @@ TEST_CASE("NelemHeuristics","[buffer]") {
   int half=buffer::n_buffer_elem_heuristics(0.5, element_size, nKQ);
   int quarter=buffer::n_buffer_elem_heuristics(0.25, element_size, nKQ);
 
-  REQUIRE_THAT(all, Catch::Matchers::WithinAbs( 2.*half, 2.));
-  REQUIRE_THAT(all, Catch::Matchers::WithinAbs(4.*quarter, 2.));
+  std::cout << "GH says: all = " << all << std::endl;
+  std::cout << "GH says: half = " << half << std::endl;
+  std::cout << "GH says: quarter = " << quarter << std::endl;
```
Comment (egull) [inline test/test_buffer.cpp]: sure you want these outputs here?

**Review by egull — COMMENTED** [review]
hesitant to approve because the mpi test fails. But the mpi test is silly. Either we can merge and right after correct the mpi test (by disabling it, this was written for debug) or we can disable the mpi test here.

two minor cleanups noted.

File: test/test_buffer.cpp (hunk 2 above)
Comment (gauravharsha) [reply test/test_buffer.cpp]: Forgot to refactor these in - one of the tests here failed on my system, so I was checking what was the issue. I will clean these out.

**Comment by gauravharsha** [comment]
Addressed the reviews. Once PR #4 is merged, I will rebase the cleanup branch and then all tests should pass.

**Pull request merged**